### PR TITLE
Display attached card pills in sent message

### PR DESCRIPTION
**Pull request by burieberry**
sent-cards

### CODE REVIEW

**Comment by backspace** [comment]
It looks like a Matrix test expectation needs updating with this 😯

File: packages/host/app/components/ai-assistant/card-pill.gts [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  Element: HTMLElement;
+  Args: { card: CardDef };
+  Blocks: { default: [] };
+}
+
+export default class AiAssistantCardPill extends Component<Signature> {
+  <template>
+    {{#if @card.id}}
+      <Pill @inert={{true}} class='card-pill' ...attributes>
+        <div class='card-title'>{{getDisplayTitle @card}}</div>
```
Comment (lukemelia) [inline packages/host/app/components/ai-assistant/card-pill.gts]: Is there a separate task to add the realm icon to this?

Comment (lukemelia) [reply packages/host/app/components/ai-assistant/card-pill.gts]: Also, should we use the atom format of the card here instead of `getDisplayTitle`?

Comment (burieberry) [reply packages/host/app/components/ai-assistant/card-pill.gts]: I added the icon to the input attachments side, I can create a ticket to handle it for the atom template. Does this mean that atom template should have the realm icon by default? Also the atom styling for card templates is slightly different from how we previously used them, but I can update them too

**Pull request merged**